src/api/RequestApi.ts

```diff
@@ -238,13 +238,13 @@ export type IInRequest = {
   /** Required - The Employee's Office */
   office: string;
   /** Required - Can only be 'true' if it is a New to USAF Civilain.  Must be 'false' if it is a 'mil' or 'ctr' */
-  isNewCivMil: "yes" | "no";
+  isNewCivMil: "yes" | "no" | "";
   /** Required - The user's previous organization.  Will be "" if isNewCiv is false */
   prevOrg: string;
   /** Required - Can only be 'true' if is a Civ/Mil.  For Ctr, will be 'false' */
-  isNewToBaseAndCenter: "yes" | "no";
+  isNewToBaseAndCenter: "yes" | "no" | "";
   /** Required - Can only be 'true' if is a Ctr.  For others it will be false */
-  hasExistingCAC: "yes" | "no";
+  hasExistingCAC: "yes" | "no" | "";
   /** Required - Will only be defined for Ctr, for others it will be undefined*/
```

Comment:
Update comment to
/** Required - Can only be 'true'/'false' if it is Contractor.  Must be '' if it is a Civilian or Military.  */